app/background/node/service.js

```diff
@@ -535,3 +642,56 @@ export async function start(server) {
   await service.configurePaths();
   server.withService(sName, methods);
 }
+
+async function hapiGet(path = '') {
+  const res = await fetch(`https://5pi.io/hsd${path}`, {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+      'Authorization': 'Basic ' + Buffer.from(`x:775f8ca39e1748a7b47ff16ad4b1b9ad`).toString('base64'),
```

Comment:
LOL why bother? ;-)

Reply:
Too lazy to change code in old 5pi 😂
New 5pi is gonna public to all by default after we switch